### PR TITLE
Tweak Issue Selector

### PR DESCRIPTION
**Pull request by Ornias1993**
### Motivation and Context
While the current issue selector makes things easier, there are a few things that could require tweaking:
- #10825 has shown us that we really need to point to the mailinglist in a more obvious fasion
- #10825 has also shown us that there is still an option to file blank issues, this should be rarely needed if at all.
- #10830 shows a case of users looking for support, picking the wrong option

### Description
- This PR adds links to both the Linux and FreeBSD mailinglists to the Issue selector
- This PR disables the link pointing to "create a blank issue" in the issue selector
- This PR also clearly labels the mailinglist for "Support", to motivate users to ask questions in their related mailinglist

Please note it still seems to be possible to file blank issues by pointing it to the right link directly, but thats not an issue directly.

### How Has This Been Tested?
- Seperate repo: https://github.com/Ornias1993/zfs/issues/new/choose

### Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Performance enhancement (non-breaking change which improves efficiency)
- [X] Code cleanup (non-breaking change which makes code smaller or more readable)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Documentation (a change to man pages or other documentation)

### Checklist:

- [ ] My code follows the ZFS on Linux [code style requirements](https://github.com/zfsonlinux/zfs/blob/master/.github/CONTRIBUTING.md#coding-conventions).
- [X] I have updated the documentation accordingly.
- [X] I have read the [**contributing** document](https://github.com/zfsonlinux/zfs/blob/master/.github/CONTRIBUTING.md).
- [X] I have added [tests](https://github.com/zfsonlinux/zfs/tree/master/tests) to cover my changes.
- [X] I have run the ZFS Test Suite with this change applied.
- [X] All commit messages are properly formatted and contain [`Signed-off-by`](https://github.com/zfsonlinux/zfs/blob/master/.github/CONTRIBUTING.md#signed-off-by).

Signed-off-by: Kjeld Schouten-Lebbing <kjeld@schouten-lebbing.nl>
Requires-builders: style

### CODE REVIEW

**Comment by ghfields** [comment]
How about adding IRC info also?
(#zfsonlinux on freenode)
(#openzfs on freenode)

**Comment by Ornias1993** [comment]
> How about adding IRC info also?
> (#zfsonlinux on freenode)
> (#openzfs on freenode)

The thing is: You don't want too many options.
I expect there already will be more channels of other platforms get merged, so I limited myself to the support channels as they are already promoted (which is primarily the mailinglist). I also didn't list all mailinglists for a reason, but only one primary support channel per platform

**Comment by ghfields** [comment]
A counterpoint is that mailing list request is "loud".  Many basic issues (as well as their followup responses and conversations) quickly add noise that can be avoided if the submitter is comfortable triaging through IRC first.  As these mailinglist increase, reducing distributed fatigue may be an important consideration.

Currently, IRC methods are promoted through the newcomer guide and included in the code of conduct.  Ultimately, with the OpenZFS restructuring, I, personally, would like to see IRC channel consolidation.  It may be OK to temporarily withhold IRC from the the Issue selector, if the OpenZFS org is looking at streamlining official IRC channels.

**Comment by Ornias1993** [comment]
@ghfields Sorry, but from an UX standpoint i'm not going to add 7 options on this screen. The current github advice when creating an issue is the mailinglist and this just adds the link to the selector (A reference thats already in the github issue commenting as an advice).

I'm all for consilidation:
There should IMHO be 1(!) OpenZFS support mailinglist, 1(!) OpenZFS support IRQ. 
Which makes it 5 options total, which is just perfect from an UX standpoint. Also I don't really like pointing FreeBSD folks to a FreeBSD mailinglist at all tbh.

So TLDR:
I agree the current convoluted support structure is bad, but I can also not fuckup UX je because of it.

**Comment by Ornias1993** [comment]
@ghfields It seems I was to harsh on you, my apologies. I was under the wrong impression you where refering to adding all sorts of platform specific IRQ channels.
It seems adding IRQ would just be one more option as there is already a consolidated #openZFS IRQ.

I've added the option for IRQ to this PR and opened an issue (#10845) about opening a multiplatform OpenZFS discuss mailinglist.

I hope this satisfies your feedback :)

**Comment by ghfields** [comment]
No worries.  While I won't take credit for having the "original idea", but I did privately suggest moving zfs repo from zfsonlinux into openzfs account well before it was mentioned in the community.  So.... I'm a proponent of the consolidation effort.  

The code contribution path looks good.  Now is probably a good time to work on the formal communications and support side as well.  This is a fine example of how we still are fragmented.  Thanks for starting that conversation.

**Comment by Ornias1993** [comment]
@ghfields The contribution path looks perfectly fine, I think it's one of the prime examples how multi-platform doesn't have to be an issue :)

But indeed, I was also amazed I there wasn't a discuss like mailinglist for openzfs as a whole... From a "user wanting help with an issue" this makes things needlessly complicated, which is a shame. Goal with this PR is to at least highlight all options (even those outside of github) to make it as painless as possible to find a suitable format for user-questions. :)

#10845 would be a great start for streamlining things even more...

**Comment by Ornias1993** [comment]
I'm going to merge this with #10853 and add the request for a clear OpenZFS-Discuss mailinglist accordingly.
